guides/advanced_index_actions.md

```diff
@@ -0,0 +1,113 @@
+# Advanced Index Actions
+
+In this guide, we will look at some advanced index actions that are not covered in the [Index Lifecycle](index_lifecycle.md) guide.
```

Comment:
That file doesn't exist yet, so the link checker breaks. You can combine all the guides in 1 PR or make them in order they link to each-other.

Reply:
@dblock  Can we wait for the index lifecycle to merge first and then re-run the link checker?